### ISSUE TITLE
Support for electric floor heating with separate sensors

### ISSUE THREAD

**Issue by FearNaBoinne** [999LV/SmartVirtualThermostat#28]
It would be nice if you could implement floor heating support, where the heating-mats are paired with specific sensors to ensure the floor temperature doesn't exceed a preset temperature range. (esp. important for wood/laminate floors).

I.e. something like [(switchIDX,sensorIDX,minTemp,maxTemp),(switchIDX,sensorIDX,minTemp,maxTemp),(switchIDX,sensorIDX,minTemp,maxTemp)]

**Comment by 999LV** [comment]
Let me rephrase your suggestion to make sure I understand... you have a temperature sensor for your floor and want to shut off heating when a certain temperature is reached, as a safety precaution ?

If this is the case, then I’d suggest to use an event script to set the thermostat pause switch on when that temperature is reached and then to off when the floor has cooled enough. Your particular setup is not common I would think and adding new parameters to the plugin just very difficult (domoticz limitation for hardware parameters has already forced me to use long comma separated values...).
Hope this helps
